<ui version="4.0" >
 <class>ShortcutsDialog</class>
 <widget class="QDialog" name="ShortcutsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>447</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Shortcuts Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QTableWidget" name="tableWidget" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>Actions</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Shortcuts</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>61</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Press any key here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editShortcut" />
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonAssign" >
       <property name="text" >
        <string>Assign</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>181</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDone" >
       <property name="text" >
        <string>Done</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonDone</sender>
   <signal>clicked()</signal>
   <receiver>ShortcutsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
